CHARTER.adoc

```diff
@@ -0,0 +1,195 @@
+
+
+
+== Business Value Proposition
+
+In the geospatial world, new cloud native data formats are emerging. Zarr is a generic data format for n-dimensional arrays that enables access to data in compressed chunks of the original array and has become increasingly popular to use for geospatial purposes. In June 2022, the OGC endorsed a community standard of Zarr V2.0 (https://zarr.readthedocs.io/en/stable/spec/v2.html). The purpose of this charter is to adopt a more explicit GeoZarr as an OGC Standard which would provide guidance to standardize the approach for encoding various aspects of geospatial data in zarrs.
+
+== Scope of Work
+
+The goal of the GeoZarr specification is to establish flexible and inclusive conventions for the Zarr cloud-native format, specifically designed to meet the diverse requirements within the geospatial domain. These conventions aim to provide a clear and standardized framework for organizing and describing data, ensuring unambiguous representation. 
+
+The objectives of GeoZarr conventions includes:
+
+1. Compatibility: Ensuring easy compatibility with popular mapping and data analysis tools such as GDAL, Xarray, ArcGIS, and QGIS, enabling seamless integration into existing workflows.
+2. Visualisation: Simplifying the creation and display of geospatial data in web browsers without the need for complex workarounds, making geospatial information more accessible to users.
+3. Dimensions: Supporting multidimensional data, such as hyperspectral and altitude information, to address diverse geospatial data requirements.
+4. Data Discovery: Providing metadata for discovering, accessing, and retrieving the data, including subsets or different arrangements of the data.
+5. Mixing Data: Facilitating the combination of different types of geospatial data, including satellite images, elevation maps, and weather models, to create comprehensive and informative datasets.
+8. Flexibilty: Allowing scientists and researchers to work with diverse data types and projections in their preferred software and programming languages, promoting flexibility and adaptability in geospatial data processing and analysis.
+
+Specifically, the convention should provide guidance to standardize the approach for encoding various aspects of geospatial data, including:
+
+* Multiple related variables with heterogeneous coordinates (e.g., children or linked datasets)
+* Multiple resolutions of the data, possibly leveraging multiscale drafts
```

Comment:
Seems like a separate issue --- suggest removing as out-of-scope (or saying something like, "Support for / compatibility with Zarr extensions, such as multi-resolution (LINK)."

Reply:
@ashiklom I'd suggest we keep `Multiple related variables with heterogeneous coordinates (e.g., children or linked datasets)` so we accurately capture the need to encode potentially different CRS information for different variables.

As you and @rabernat discussed, multi-resolution is not specific to domain so let's move this concept to a new Zarr extension.

Reply:
Disagreed. 
The inclusion of multiscales is a crucial aspect in enabling fast visualization (as for COG). 
In the geospatial domain, specific attributes may be utilized for describing multiscales (as outlined in the draft), and it would typically be linked to OSM levels.

The specification should  encompass all pertinent aspects instead of excluding potentially more generic ones. If a candidate convention emerges that adequately fulfills all needs, it can then be updated to a reference within the specification.

Reply:
It can be useful to develop these approaches with a specific use case in mind. I'd suggest leaving this in scope here but planning to work the problem in parallel in a more general context.

Reply:
I am bringing this up for discussion at our bi-weekly meeting. It's just one of a few examples where the need is actually upstream, but right now there is no champion for implementation.

Reply:
We just talked about this at the meeting, and multi-resolution (and for that matter multiple chunking schemes) sound like they should be their own (optional) conformance classes.  COG is a good example to look at where overviews are a separate, thus optional, conformance/requirements class.

Reply:
@briannapagan , @matthewhanson  : I have the same feeling (conformance classes for multi-resolution and a large set of other topics). However, I believe this is not the right time to discuss those aspect: when draft covnentions will be ready for a large set of topics, then we will be able to discuss how make a consistent spec.

Reply:
As in previous resolved conversation I have added the lines - which I hope is a good middle ground for all:

> Specifically, the convention should provide guidance to standardize the approach for encoding various aspects of geospatial data, including, for example the following list of potential conformance classes:

> Part of the effort of this working group is to determine what should be kept as a part of the GeoZarr core requirements, and what aspects of geospatial data should be kept as separate conformance classes, which may or may not be applicable to specific sub-domains of the geospatial community.